data_steward/analytics/rt_cdr_qc/cdr_deid_qa_report7_cope_survey.py

```diff
@@ -55,26 +63,58 @@
 # 903632,702686,715714, 715724, 715725, 715726, 1310054, 1310058, 1310066, 1310146, 1310147, 1333234, 1310065,
 #
 # 596884, 596885, 596886, 596887, 596888, 596889, 1310137,1333016,1310148,1310145,1310144
+
 # From DC-2109
 # 765938,765939, 765940, 765941, 765942, 765943, 765944, 765945, 765946, 765947, 765948, 765949, 765950, 765951, 765952
 
+# DC-2113
+# RT suppression
 # +
+
+rt_concepts_dict = {
```

Comment:
This dictionary is a great idea! It looks cleaner and more intuitive.

Reply:
Thanks, this was a mess to figure out which concept came from which ticket.